src/apply.jl

```diff
@@ -110,7 +109,9 @@ is appended to `A` along the `append_dim` dimension. The remaining `kwargs` corr
 the usual [`Transform`](@ref) being invoked.
 """
 function apply_append(A::AbstractArray, t; append_dim, kwargs...)::AbstractArray
-    return cat(A, apply(A, t; kwargs...); dims=append_dim)
+    result = apply(A, t; kwargs...)
+    result = _postformat(cardinality(t), result, A, append_dim)
```

Comment:
Just want to check that `append_dim` is definitely right. This came up in the `apply_append` PR but do we have any test cases for `append_dim != dims` yet?